plugins/PrivacyManager/lang/en.json

```diff
@@ -33,7 +33,7 @@
         "DoNotTrack_Description": "Do Not Track is a technology and policy proposal that enables users to opt out of tracking by websites they do visit, including analytics services, advertising networks, and social platforms.",
         "DoNotTrack_Disable": "Disable Do Not Track support",
         "DoNotTrack_Disabled": "Matomo is currently tracking all visitors, even when they specified \"I do not want to be tracked\" in their web browsers.",
-        "DoNotTrack_DisabledMoreInfo": "We recommend to respect your visitors privacy and enable DoNotTrack support.",
+        "DoNotTrack_DisabledMoreInfo": "We recommend respecting your visitors privacy and enable DoNotTrack support.",
```

Comment:
"visitors" needs a trailing apostrophe (to indicate possession) and the sentence seems quite awkward. How about:
`We recommend enabling DoNotTrack support to respect your visitors' privacy`

Reply:
Sounds good. I think I saw more cases where possessive apostrophes were suggested, but I am quite insecure about placing them correctly (as they aren't used in German), so I didn't change them.